src/Models/CoursesModel.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace App\Models;
+
+
+use PDO;
+
+class CoursesModel
+{
+    protected PDO $db;
+
+    // Here, the parameter is automatically supplied by the Dependency Injection Container based on the type hint
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
```

Comment:
Another thing to think about here - I've added this example model which takes a PDO connection, but the issue there is that students will need to connect slim to a DB straight away.

Reply:
oh just realised the modal doesnt actually use the DB, so they only need to have a DB with he right name, which isn't too bad

Reply:
perhaps we could be sneaky and when we get them to do the SQL excercises in week 2 (adults/children etc..) we could make that named example. That way we could pre-emptively solve the prob before it arises?